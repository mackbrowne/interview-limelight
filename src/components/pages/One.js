// Framework
import React from 'react';

// Components
import { Container, Row, Col } from 'reactstrap';

/**
 * Page 1 page for the app!
 *
 * @param {Object} props All the properties passed to the React Component.
 * @returns {StatelessComponent} Stateless functional React component.
 */
const PageOne = props => (
  <Container fluid>
    <Row>
      <Col>
        <p>Page 1</p>
      </Col>
    </Row>
  </Container>
);

export default PageOne;
