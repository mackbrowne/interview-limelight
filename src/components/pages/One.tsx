// Framework
import React from 'react';

// Components
import { Container, Row, Col } from 'react-bootstrap';

/**
 * Page 1 page for the app!
 *
 * @param {Object} props All the properties passed to the React Component.
 * @returns {StatelessComponent} Stateless functional React component.
 */
export default function PageOne() {
  return (
    <Container fluid>
      <Row>
        <Col>
          <p>Page 1</p>
        </Col>
      </Row>
    </Container>
  );
}
