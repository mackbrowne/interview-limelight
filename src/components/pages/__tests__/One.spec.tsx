import React from 'react';
import { mount } from 'enzyme';

import One from '../One';

describe('<One /> Snapshot', () => {
  it('renders snapshot with no props', () => {
    const wrapper = mount(<One />);
    expect(wrapper).toMatchSnapshot();
  });
});
