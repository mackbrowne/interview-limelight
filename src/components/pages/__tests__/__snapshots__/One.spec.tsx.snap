// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<One /> Snapshot renders snapshot with no props 1`] = `ReactWrapper {}`;
